core/src/main/scala/org/scalarelational/column/property/NotNull.scala

```diff
@@ -1,8 +0,0 @@
-package org.scalarelational.column.property
-
-/**
- * @author Matt Hicks <matt@outr.com>
- */
-object NotNull extends ColumnProperty {
```

Comment:
Can we make this a bit less invasive?  What if we leave NotNull but the property gets implicitly added if the type is "Option"?  That gives the flexibility if you actually want to use null instead of Option as well as closer representation to actual SQL.  Finally, it gives you the ability to write condition checks in more complex forms for NOT NULL and NULL.